services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -560,7 +591,10 @@ export class NOWPermitGeneration extends Component {
               <Popconfirm
                 placement="bottomRight"
                 title="You have unsaved changes. Are you sure you want to cancel?"
-                onConfirm={this.handleCancelDraftEdit}
+                onConfirm={() => {
+                  this.handleCancelDraftEdit();
+                  this.props.storeEditingPreambleFlag(false);
+                }}
```

Comment:
'this.props.storeEditingPreambleFlag(false);' - This doesn't seem relevant to this button.